src/library_fs.js

```diff
@@ -1044,7 +1033,7 @@ FS.staticInit();
     utime(path, atime, mtime) {
       var lookup = FS.lookupPath(path, { follow: true });
       var node = lookup.node;
-      node.node_ops.setattr(node, {
```

Comment:
Previously there was no error handling for `setattr` being missing, so if we called `utime` on a read only node it would crash.